Sprint-1/implement/sum.test.js

```diff
@@ -13,24 +13,47 @@ const sum = require("./sum.js");
 // Given an empty array
 // When passed to the sum function
 // Then it should return 0
-test.todo("given an empty array, returns 0")
+test("Given an empty array, should return 0", function() {
+    const result = sum([]);
+    expect(result).toEqual(0)
+});
 
 // Given an array with just one number
 // When passed to the sum function
 // Then it should return that number
+test("Given an array with one number, should return the same number", function() {
+    const result = sum([5]);
+    expect(result).toEqual(5);
+  });
 
 // Given an array containing negative numbers
 // When passed to the sum function
 // Then it should still return the correct total sum
+test("Given an array with negative numbers, should return the correct total sum", function() {
+    const result = sum([10, -5, 15, -10]);
+    expect(result).toEqual(10);
+  });
 
 // Given an array with decimal/float numbers
 // When passed to the sum function
 // Then it should return the correct total sum
+test("Given an array with decimal numbers, should return the correct total sum", function() {
+    const result = sum([10.5, 20.3, -5.2, 5]);
+    expect(result).toEqual(30.6); // 10.5 + 20.3 - 5.2 + 5 = 30.6
```

Comment:
Decimal numbers in most programming languages (including JS) are internally represented in "floating point number" format. Floating point arithmetic is not exact. For example, the result of `46.5678 - 46 === 0.5678` is false because `46.5678 - 46` only yield a value that is very close to `0.5678`.

So `expect(sum([1.1, 1.1, 1.1])).toEqual(3.3);` would probably fail.

Can you find a more appropriate way to test a value (that involves decimal number calculations) for "equality?